src/node_worker.h

```diff
@@ -34,14 +55,13 @@ class Worker : public AsyncWrap {
     tracker->TrackFieldWithSize(
         "isolate_data", sizeof(IsolateData), "IsolateData");
     tracker->TrackFieldWithSize("env", sizeof(Environment), "Environment");
-    tracker->TrackField("thread_exit_async", *thread_exit_async_);
```

Comment:
We’re not tracking the `uv_async_t` anymore, right? Maybe we should add something like `tracker->TrackInlineField()` that allows us to keep track of `MemoryRetainer`s that are direct members of the class…

Reply:
you mean - the one represented by `thread_exit_async_` ? that is replaced with`AsyncRequest` objects that creates `uv_async_t` objects, and tracks through the interface method. the `async_` field in `AsyncRequest` is still a pointer,  direct member of neither `AsyncRequest` nor `Worker`.

Reply:
Neither `*thread_stopper_.async_` nor `*on_thread_finished_.async_` are tracked, yes, because we don’t inform the `tracker` about the existence of the `AsyncRequest` fields.

Also, side note: I’m just noticing that we have the `IsolateData` and `Environment` fields listed here as well, which I’m not sure makes sense given that they are no longer directly allocated by this object…

Reply:
sorry, I don't understand. `*thread_stopper_.async` and `*on_thread_finished_.async_` are not tracked through the tracker instance or worker, but those are tracked through the tracker instance of `AsyncRequest` object (line 98):
```c++
void AsyncRequest::MemoryInfo(MemoryTracker* tracker) const {
  Mutex::ScopedLock lock(mutex_);
  if (async_ != nullptr) tracker->TrackField("async_request", *async_);
}
```
Isn't it enough? I hope we don't need multiple trackers for the same allocation?

For the `IsolateData` and `Environment`: I just removed those from being actively tracked by the worker and pushed in under this PR itself.

Reply:
@gireeshpunathil The problem is that the memory tracker doesn’t know that it should call `AsyncRequest::MemoryInfo`. Currently, the way to inform it would be adding `tracker->TrackField("thread_stopper_", &thread_stopper_);`, but then we would end up tracking the memory for the `AsyncRequest` itself twice.

Reply:
@gireeshpunathil Should we change this PR to use `TrackInlineField` now?

Reply:
@addaleax - yes. Though I knew this depend on #26161 for a moment I forgot about that!